benchmarking/evaluations/scenarios/diversify_social_only.py

```diff
@@ -0,0 +1,21 @@
+from typing import List
+
+from benchmarking.evaluations.enums import PreferenceDirection, PreferenceSubject
+from benchmarking.evaluations.interfaces import (
+    Scenario,
+    Goal,
+)
+from benchmarking.evaluations.goals import WeightGoal, PreferenceGoal
+
+
+class DiversifySocialFriends(Scenario):
```

Comment:
I'm not sure the name of this describes the scenario still.

From the goals you've defined, it seems like this is `IncludeSocialFriends`

And the `def name()` should be more like "Create teams that group social friends together"
(the name should be descriptive)